docs/quickstart.md

```diff
@@ -1,68 +1,334 @@
+5. Enter the display name **Harvest**.
+6. Set the role to **Editor**.
+7. Click on **Create**. The service account will appear in the dashboard.
+8. Navigate back to **Service Account**.
+9. Click on **Add service account token** for the Harvest service account.
+10. Click on **Generate Token**.
+11. Click on **Copy to clipboard and close**.
+
+**IMPORTANT:** This is the only opportunity to save the token. Immediately paste it into a text file and save it. The token will be needed by Harvest later on.
+
+For detailed instructions, please refer to the [Grafana API Keys documentation](https://grafana.com/docs/grafana/latest/administration/api-keys/).
+
+### 10. Import Grafana Dashboards
+
+To import Grafana dashboards, use the following command:
+
+```bash
+cd ${HARVEST_INSTALL_PATH}/harvest-${HARVEST_VERSION}-1_linux_amd64
+bin/harvest grafana import --addr localhost:3000
+```
+
+You will be prompted to enter the Grafana API token. Paste the token you generated in the previous step.
+
+```bash
+#### You will be requested to 'Enter API Token'
+#### Enter the token saved before
+#### You will be asked to save token for later use, enter 'Y'
+
+### If you have removed and re-installed Grafana, you will get an error telling that current API Key is not valid any more
+### You need to enter the new API Key
+
+using API token from config
+.error connect: (401 - 401 Unauthorized) Unauthorized
+enter API token:
+```
 
-Note: the current dashboards specify Prometheus as the datasource. 
-If you use the InfluxDB exporter, you will need to create your own dashboards.
+It will take a few seconds, but at the end, all dashboards will be imported.
 
-## 5. Verify the metrics
+### 9. Verify Dashboards in Grafana
 
-If you use a Prometheus Exporter, open a browser and navigate to [http://0.0.0.0:12990/](http://0.0.0.0:12990/)
-(replace `12990` with the port number of your poller). 
-This is the Harvest created HTTP end-point for your Prometheus exporter. 
-This page provides a real-time generated list of running collectors and names of exported metrics.
+After adding the Prometheus datasource, you can verify that your dashboards are correctly displaying data. Follow these steps:
 
-The metric data that is exported for Prometheus to scrap is 
-available at [http://0.0.0.0:12990/metrics/](http://0.0.0.0:12990/metrics/). 
+1. Open your web browser and navigate to `http://localhost:3000` or `http://<machine-ip>:3000`
+2. Enter your Grafana credentials to log in. The default username is `admin` and the default password is also `admin` (you may have changed this during setup).
+3. Click on the "three lines" button (also known as the hamburger menu) in the top left corner of the Grafana interface. From the menu, select **Dashboards**.
+4. In the Dashboards section, find and click on the **Volume Dashboard**. This dashboard should have been pre-configured or imported as part of your setup. Once the Volume Dashboard is open, you should see various panels displaying data
```

Comment:
Our dashboard links should be deterministic now, which means we can link directly like so:

4. Open the [Volume](http://localhost:3000/d/cdot-volume/ontap3a-volume?orgId=1) dashboard. Once the dashboard opens, you should see volume data displayed.